### PR TITLE
Spell check

### PR DESCRIPTION
**Pull request by sungam3r**
Just spell check

### CODE REVIEW

**Comment by sungam3r** [comment]
[clabot:check]

File: examples/ConfluentCloud/Program.cs [hunk 1]
```diff
@@ -28,7 +28,7 @@ namespace ConfluentCloudExample
     ///     
     ///     https://www.confluent.io/confluent-cloud/
     /// 
-    ///     Confluent Cloud does not auto-create topics. You will need to use the ccloud
+    ///     Confluent Cloud does not auto-create topics. You will need to use the cloud
```
Comment (mhowlett) [inline examples/ConfluentCloud/Program.cs]: this should be `ccloud`

**Comment by mhowlett** [comment]
thanks @sungam3r! just one place that isn't actually a spelling error. Can you:
1. change `cloud` back to `ccloud`.
2. re-target to the 1.0.x branch (not master).

then this is good to merge.

File: examples/ConfluentCloud/Program.cs (hunk 1 above)
Comment (sungam3r) [reply examples/ConfluentCloud/Program.cs]: done with retargeting to 1.0.x branch

File: Confluent.Kafka.sln [hunk 2]
```diff
@@ -5,51 +5,58 @@ VisualStudioVersion = 15.0.26124.0
 MinimumVisualStudioVersion = 15.0.26124.0
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{1EFCD839-0726-4BCE-B745-1E829991B1BC}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Confluent.Kafka", "src\Confluent.Kafka\Confluent.Kafka.csproj", "{09C3255B-1972-4EB8-91D0-FB9F5CD82BCB}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Confluent.Kafka", "src\Confluent.Kafka\Confluent.Kafka.csproj", "{09C3255B-1972-4EB8-91D0-FB9F5CD82BCB}"
```
Comment (mhowlett) [inline Confluent.Kafka.sln]: don't want all these GUID changes...

Comment (sungam3r) [reply Confluent.Kafka.sln]: done

**Comment by mhowlett** [comment]
awesome, thanks!

**Pull request merged**